src/components/About/AboutPage.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { } from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import { Image, CardDeck } from 'react-bootstrap'
+import '../../styles/AboutPage.css'
+import '../../styles/App.css'
+import House from '../../assets/BrfLemansgatan.jpeg'
+import codeStock from '../../assets/stockCodePic.jpeg'
+import Hello from '../../assets/hello.jpg'
```

Comment:
Finns det något bättre sätt än att ha import för varje bild tro? Inte för det spelar roll men kanske är snyggare bara

Reply:
Kanske att man har const codeStock = require('filepath'). Det kanske är lika fult dock! för mig spelar det ingen roll!

Reply:
I think that it is good. Then we can know statically what files are required to run the program.

Reply:
har kvar det som det är!